tests/product/standalone/test_installation.py

```diff
@@ -70,7 +69,7 @@ def test_install_non_root(self):
     @docker_only
     def test_install_on_wrong_os_offline_installer(self):
         image = 'teradatalabs/ubuntu-trusty-python2.6'
-        tag = BASE_IMAGES_TAG
```

Comment:
you can pull docker image here so you then, you could remove `make docker-images`

Reply:
not 100% sure leaving for now until it works